test/system/update_patient_info_test.rb

```diff
@@ -1,6 +1,8 @@
 require 'application_system_test_case'
 
 class UpdatePatientInfoTest < ApplicationSystemTestCase
+  extend Minitest::OptionalRetry
```

Comment:
this just kept popping up here and there with random failures. this seems to improve.